client/src/elements/molecules/item-list-entry.js

```diff
@@ -26,12 +30,34 @@ export class ItemListEntry {
     });
   }
 
+  toggleOptionDropdown() {
+    this.optionsDropdown.classList.toggle(
+      "item-list-entry__options-dropdown--show"
+    );
+  }
+
+  activateItem() {
+    this.item.activate();
+  }
+
+  deactivateItem() {
+    this.item.deactivate();
+  }
+
   deleteItem() {
     this.item.delete().then(() => {
```

Comment:
this should first open a confirmation dialog and only delete after confirmation. You can see an example of how to use it here: https://github.com/nzzdev/Q-editor/blob/dev/client/src/pages/item-overview.js#L107-L115

Reply:
Added in https://github.com/nzzdev/Q-editor/pull/133/commits/f653a255ba161bb98e01d0f914ddbf204da4bd41. Modal still has to be redesigned.